Add first draft of contract

mmlkrx opened the pull request:
@Henryvw quick first draft, please add corrections or let me know what I forgot.

mmlkrx committed:
Add first draft of contract

Henryvw commented:
Roger Will do 

Sent from my iPod

> On Oct 20, 2017, at 12:35 PM, Matthias Loker <notifications@github.com> wrote:
> 
> @Henryvw quick first draft, please add corrections or let me know what I forgot.
> 
> You can view, comment on, or merge this pull request online at:
> 
>   https://github.com/Henryvw/eruditus/pull/1
> 
> Commit Summary
> 
> Add first draft of contract
> File Changes
> 
> A contract.md (43)
> Patch Links:
> 
> https://github.com/Henryvw/eruditus/pull/1.patch
> https://github.com/Henryvw/eruditus/pull/1.diff
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub, or mute the thread.
>

Henryvw committed:
Added Game Master commitment & rules to contract

- Capitalized all proper nouns / terms (optional, we can go back to lower-case if you prefer)
- Added 'Presentation Session' as a term
- Added 'Learning Recap' as a term

Henryvw committed:
added end to code block for game rules

Henryvw committed:
Changed "wrap-up" to "learning recap" for term consistency

Henryvw committed:
changed learning recap definition to 15 minutes

mmlkrx committed:
Small changes

mmlkrx committed:
Change contract name
